src/main/kotlin/org/jitsi/videobridge/octo/config/OctoConfig.kt

```diff
@@ -21,12 +21,25 @@ import org.jitsi.config.LegacyFallbackConfigProperty
 import org.jitsi.config.legacyConfigAttributes
 import org.jitsi.config.newConfigAttributes
 import org.jitsi.utils.config.FallbackProperty
+import org.jitsi.utils.config.SimpleProperty
 import org.jitsi.utils.config.exception.ConfigValueParsingException
 
 class OctoConfig {
     class Config {
         companion object {
 
+            class QueueSizeProperty : SimpleProperty<Int>(
+                    newConfigAttributes {
+                        name("videobridge.octo.q-size")
```

Comment:
I'd spell out "queue" in the parameter names.

Reply:
Perhaps also add "receive" to the name. We currently don't have a send queue, but we might add one in the future.